nih_plug_egui/Cargo.toml

```diff
@@ -21,7 +21,7 @@ raw-window-handle = "0.5"
 baseview = { git = "https://github.com/RustAudio/baseview.git", rev = "45465c5f46abed6c6ce370fffde5edc8e4cd5aa3" }
 crossbeam = "0.8"
 egui-baseview = { git = "https://github.com/BillyDM/egui-baseview.git", rev = "68c4d0e8e5c1c702a888a245f4ac50eddfdfcaed", default-features = false }
-lazy_static = "1.4"
+once_cell = "1.19"
```

Comment:
[`LazyLock`](https://doc.rust-lang.org/std/sync/struct.LazyLock.html) is now the standard library now, so instead of swapping out `lazy_static` for `once_cell` it's probably better to just bump the MSRV and use that instead.

Reply:
Oh yeah it is, I'll double check which version it was added in. I always get them mixed up